src/main/java/ch/njol/skript/effects/EffUnlimitedLifetime.java

```diff
@@ -0,0 +1,68 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Unlimited Lifetime")
+@Description("Makes a dropped item have an unlimited lifetime.")
+@Examples({
+	"drop tripwire hook at player's location",
+	"make last dropped item have unlimited lifetime"})
+public class EffUnlimitedLifetime extends Effect {
```

Comment:
I wonder if instead of having this as an effect we should just allow `set lifespan of last dropped item to infinity`

up for debate ofc

Reply:
is infinity a valid timespan? thats the only reason i added it as an effect

Reply:
no I just like thinking about numbers apparently

if `forever` is valid syntax then this stands - if not please ignore me 🤣

Reply:
i like the idea, i just feel like `set lifespan of last dropped item to forever` doesn't sound right??? it could just be me but im willing to change it if its valid

Reply:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/72931234/194788459-f26d236f-bf2c-4d1c-ae51-9de17039139c.png">
I dont think `forever` is a valid timespan but i'll do another check and let you know

Reply:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/72931234/194788564-4693e66a-20e3-4f85-8034-a305002caea8.png">
Yeah it isn't, `set {_1 second} to 1 second` parsed fine

Reply:
It might be worth adding one for this (and other timestamp needs.)

Reply:
we can use `infinite lifetime state` as a boolean

Reply:
I suppose we could go with something like
```
make %entities% have [an] (unlimited|infinite) life(time|span) # what we have now
# but also add
make %entities% live forever
```
idk just out loud thinking :)